lib/addons/p5.dom.js

```diff
@@ -1043,281 +1148,4 @@
       p5.prototype.set.call(this, x, y, imgOrCol);
     }
   };
```

Comment:
hm it seems like these changes probably got pulled into your repo at some point but then lost in a merge or something. that's why when you pull from upstream it's not registering anything new. i'm not sure the best thing to do here, but it might be to manually add them back in.